packages/pds/src/db/index.ts

```diff
@@ -186,13 +187,16 @@ export class Database {
     }
   }
 
-  async getUser(usernameOrDid: string): Promise<User | null> {
-    let query = this.db.selectFrom('user').selectAll()
+  async getUser(usernameOrDid: string): Promise<(User & UserDid) | null> {
+    let query = this.db
+      .selectFrom('user')
+      .innerJoin('user_did', 'user_did.username', 'user.username')
+      .selectAll()
     if (usernameOrDid.startsWith('did:')) {
       query = query.where('did', '=', usernameOrDid)
     } else {
       query = query.where(
-        sql`lower(username)`,
+        sql`lower(user.username)`,
```

Comment:
this lines giving me hell. works fine with sqlite but not pg

with "username" it gives the error
`column reference \"username\" is ambiguous`

with "user.username" it gives the error
`syntax error at or near \".\""`